engine/rpcserver.go

```diff
@@ -742,7 +742,42 @@ func (s *RPCServer) GetOrders(ctx context.Context, r *gctrpc.GetOrdersRequest) (
 
 // GetOrder returns order information based on exchange and order ID
 func (s *RPCServer) GetOrder(ctx context.Context, r *gctrpc.GetOrderRequest) (*gctrpc.OrderDetails, error) {
-	return &gctrpc.OrderDetails{}, common.ErrNotYetImplemented
+	exch := GetExchangeByName(r.Exchange)
+	if exch == nil {
+		return nil, errors.New("exchange is not loaded/doesn't exist")
+	}
+	result, err := exch.GetOrderInfo(r.OrderId)
```

Comment:
Please check the error

Reply:
I am never sure what is the best practice to return errors. I used `fmt.ErrorF` as I wanted to decorate the `error` message.

Reply:
fmt.Errorf is fine. The only thing to ever watch out for is whether the error you're outputting is also highly decorated. Most implementations of wrapper functions will return an error along the lines of `%exchangename% - failed to retreive %orderID% %error%.